maintainers/maintainer-list.nix

```diff
@@ -6928,6 +7040,7 @@
   };
   maxeaubrey = {
     email = "maxeaubrey@gmail.com";
+    matrix = "@maxeaubrey:matrix.org";
```

Comment:
Ditto on @endocrimes' comment. This should be treated as opt-in for multitudes of reasons, whether that's usefulness of information, or someone just not wanting to collate their various accounts and identities.